fdbserver/workloads/Mako.actor.cpp

```diff
@@ -41,7 +41,7 @@ struct MakoWorkload : TestWorkload {
 	// used for periodically tracing
 	std::vector<PerfMetric> periodicMetrics;
 	// store latency of each operation with sampling
-	std::vector<ContinuousSample<double>> opLatencies;
+	std::vector<DDSketch<double>> opLatencies;
```

Comment:
We should just move from double to microseconds for latencies.  2^32 microseconds is over an hour.